README.md

```diff
@@ -4,6 +4,7 @@
 
 #  Storybook Addon Material-UI
 
+## A big thanks to Alex Panov and all the devs previously who contributed to this project. This is primarily a republished package of `storybook-addon-material-ui-5`
```

Comment:
Well, thank you, but that on my own page would look like ass kiss :D
Can you add yourself in there @mw-sezzle? =)

Reply:
I've cleaned this up